src/app/src/main/kotlin/org/imperial/mrc/hint/utils/SecurePaths.kt

```diff
@@ -0,0 +1,21 @@
+package org.imperial.mrc.hint.utils
+
+enum class SecurePaths{
+
+    ADD{
+        override fun pathList(): List<String>
+        {
+            return listOf("/adr/**","/user/**","/project/**")
```

Comment:
Sorry, this probably wasn't in the ticket, but I think we should add /projects too, that's the route to fetch all projects for a user.

Reply:
@EmmaLRussell I thought this was the route that just returns the Index and should be excluded here?

Reply:
In fact just looking at the code I'm now very confused - do we have 2 routes `projects/` and `projects` that return different things based only on the presence of a trailing slash? That seems... bad!

Reply:
I suggest we exclude from this ticket and open a new ticket to change this